edx_lint/pylint/annotations_check.py

```diff
@@ -362,15 +370,50 @@ class FeatureToggleAnnotationChecker(AnnotationBaseChecker):
             NON_BOOLEAN_DEFAULT_VALUE,
             "Feature toggle default values must be boolean",
         ),
+        ("E%d64" % BASE_ID): (
+            "missing feature toggle annotation",
+            MISSING_ANNOTATION,
+            (
+                "When a WaffleFlag/Switch object is created, a corresponding annotation must be present above in the"
+                " same module and with a matching name",
+            )
+        ),
+        ("E%d65" % BASE_ID): (
+            "invalid Django Waffle import",
+            INVALID_DJANGO_WAFFLE_IMPORT,
+            (
+                "It is not allowed to directly access Django Waffle objects and methods. Instead, import from"
```

Comment:
Consider: "Do not directly..."

Reply:
Right! I was struggling to find the right wording.